lib/ui/LandingPage.dart

```diff
@@ -0,0 +1,177 @@
+  final TextStyle detailTextStyle = const TextStyle(
+      fontSize: 17,
+      height: 1.1,
+      fontWeight: FontWeight.normal,
+      color: Color(0xff4c4e6e));
+  final TextStyle footerTextStyle = const TextStyle(
+      fontWeight: FontWeight.normal, fontSize: 15, color: Color(0xff4c4e6e));
+  final TextStyle actionTextStyle = const TextStyle(
+      fontWeight: FontWeight.w500, fontSize: 17, color: Color(0xffffffff));
+
+  final int detailTextMaxLines = 3;
+  final int footerTextMaxLines = 1;
+
+  const _DefaultStyle({
+    TextStyle detailTextStyle,
+    TextStyle footerTextStyle,
+    int detailTextMaxLines,
+    int footerTextMaxLines,
+  });
+}
+
+const LandingPageStyle _defaultStyle = const _DefaultStyle();
+
+class LandingPage extends StatelessWidget {
+  final LandingPageViewModel _viewModel;
+  final LandingPageStyle _style;
+
+  const LandingPage(
+      {Key key,
+      LandingPageViewModel viewModel,
+      LandingPageStyle style = _defaultStyle,})
+      : this._viewModel = viewModel,
+        this._style = style,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: _Constants.generalPadding,
+      child: Column(
+        children: <Widget>[
+          Flexible(
+            flex: _Constants.imageFlexValue,
+            fit: FlexFit.loose,
+            child: Container(
+              child: Image.asset(
+                _viewModel.headerImage,
+                width: double.infinity,
+              ),
+            ),
+          ),
+          Container(
+            height: _Constants.bottomContainerHeight,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: <Widget>[
+                    Image.asset(_viewModel.subtitleImage),
+                    SizedBox(height: _Constants.subtitleLineSpace,),
+                    Text(
+                      Intl.message(_viewModel.detailText),
+                      style: _style.detailTextStyle,
+                      maxLines: _style.detailTextMaxLines,
+                      overflow: TextOverflow.ellipsis,
+                      textAlign: TextAlign.center,
+                    ),
+                  ],
+                ),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: <Widget>[
+                    RoundedButton(
+                      viewModel: RoundedButtonViewModel(
+                          title: Intl.message(_viewModel.actionText),
+                          onTap: () {},),
+                      style: RoundedButtonStyle.largeRoundedButtonStyle()
+                          .copyWith(buttonTextStyle: _style.actionTextStyle),
+                    ),
+                    SizedBox(height: _Constants.actionLineSpace,),
+                    GestureDetector(
```

Comment:
Give more tap area not just the text. Remember the big fingers

Reply:
Done ! Sausage finger test passed !! xD

Reply:
cool